pkgs/by-name/sm/sm64baserom/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  requireFile,
+  runCommand,
+
+  region ? "us",
+  showRegionMessage ? true,
+}:
+# nixpkgs assumes that a file derivation is a setup script and tries to load it, so we have to put this in a directory
```

Comment:
for anyone else surprised by this, it seems to be [this](https://github.com/Mic92/nixpkgs-review/issues/407) issue. or even just `nix-shell -p sm64baserom`. this seems like a reasonable workaround.